src/dosext/mfs/mfs.c

```diff
@@ -4955,3 +4991,116 @@ static int dos_fs_redirect(struct vm86_regs *state)
 
   return ret;
 }
+
+static uint16_t check_is_our_handle(sft_t* p_sft)
```

Comment:
See LFN_A6_HELPER_OFF in bios.s.
It does similar thing, but then calls int2f.
It is important because DOS (fdpp or
freedos) may one day implement that.
And it will call to int2f/11xx.
This is how we added 0x71a6 to fdpp.